packages/mobile/src/screens/audio-screen/ChallengeRewards.tsx

```diff
@@ -145,7 +219,47 @@ export const ChallengeRewards = () => {
       {userChallengesLoading && !haveChallengesLoaded ? (
         <LoadingSpinner style={styles.loading} />
       ) : (
-        rewardsPanels
+        <Flex gap='2xl'>
+          {isRewardsCooldownEnabled ? (
+            <Paper shadow='flat' border='strong' p='l' gap='m'>
+              <Flex direction='row' justifyContent='flex-start' gap='s'>
+                {claimableAmount > 0 ? (
+                  <IconTokenGold height={24} width={24} />
+                ) : null}
+                <Text variant='heading' color='accent' size='s'>
+                  {messages.totalReadyToClaim}
+                </Text>
+              </Flex>
+              <View style={styles.pillContainer}>
+                <Text style={[styles.pillMessage, styles.readyToClaimPill]}>
+                  {cooldownAmount} {messages.pending}
+                </Text>
+              </View>
+              <Text size='s' variant='body'>
+                {claimableAmount > 0
+                  ? `${claimableAmount} ${messages.available} ${messages.now}`
+                  : messages.availableMessage(
+                      formatCooldownChallenges(cooldownChallenges)
+                    )}
+              </Text>
+              {claimableAmount > 0 ? (
```

Comment:
this works, i could also see us having a single Button, with messages and variant terneries